x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/deployment_type_select.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiComboBox,
+  EuiToolTip,
+  EuiFormRow,
+  EuiIcon,
+  type EuiComboBoxOptionOption,
+  EuiDescribedFormGroup,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { i18n } from '@kbn/i18n';
+import { CIS_INTEGRATION_INPUTS_MAP } from '../../../common/constants';
+
+export type InputType = keyof typeof CIS_INTEGRATION_INPUTS_MAP;
+
+interface Props {
+  type: InputType;
+  /**
+   * Input will be disabled when onChange is undefined
+   */
+  onChange?: (type: InputType) => void;
+}
+
+const kubeDeployOptions: Array<EuiComboBoxOptionOption<InputType>> = [
+  {
+    value: 'cloudbeat/vanilla',
+    label: i18n.translate(
+      'xpack.csp.createPackagePolicy.stepConfigure.integrationSettingsSection.vanillaKubernetesDeploymentOption',
+      { defaultMessage: 'Unmanaged Kubernetes' }
+    ),
+  },
+  {
+    value: 'cloudbeat/eks',
+    label: i18n.translate(
+      'xpack.csp.createPackagePolicy.stepConfigure.integrationSettingsSection.eksKubernetesDeploymentOption',
+      { defaultMessage: 'EKS (Elastic Kubernetes Service)' }
+    ),
+  },
+];
+
+const KubernetesDeploymentFieldLabel = () => (
+  <EuiToolTip
+    content={
+      <FormattedMessage
+        id="xpack.csp.createPackagePolicy.stepConfigure.integrationSettingsSection.kubernetesDeploymentLabelTooltip"
+        defaultMessage="Select your Kubernetes deployment type"
```

Comment:
@tinnytintin10 

do we want to change this tooltip? (for deployment type selector)